js/pagination.jsx

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2014, Trevor Landau <>
```

Comment:
Can you point me to the original source of this? Although it seems like a free license, I still need to document this somewhere.